FAQ.md

```diff
@@ -2,11 +2,151 @@
+
+## How much it costs to run BTCPay Server?
+
+BTCPay is a free software. You can run it as a self-hosted solution on your own local server, or use a hosting provider, which is what a majority of users do. Additionally, you can use a third-party BTCPay host and attach your store to it.
+
+For a self-hosted option, the price depends on a hosting provider of your choice. Be sure to choose the VPS which meets the minimal requirements.
+
+The low-priced VPS servers which community tested with the BTCPay start at around [9$/month](https://medium.com/@BtcpayServer/hosting-btcpayserver-on-lunanode-bf9ef5fff75b).
+
+The Microsoft Azure servers are slightly [easier to deploy](https://github.com/btcpayserver/btcpayserver-doc/blob/master/AzureDeployment.md), but that comfort comes with a higher price-tag of roughly 65$/month. The price for the Azure servers can be reduced to approximately 25$/month after the initial sync with [penny-pinching](https://github.com/btcpayserver/btcpayserver-doc/blob/master/AzurePennyPinching.md).
+
+### Why do I need a VPS? Can't I just run BTCPay on my home computer?
+
+Just like you can host a website locally, you can do it with BTCPay. Please be aware that your PC would have to meet the minimal requirements and also run 24/7 if you don't want interruptions. While local hosting is good for testing, for production it's really not a viable solution.
+
+### Can the cost of running BTCPay on Microsoft Azure be reduced?
+
+Yes. Currently, the one-click deployment requires Azure Virtual Machine, but if you’re a programmer or understand VM’s you can deploy one on a different VM. Furthermore, the changes in the Bitcoin Core 0.16.0 (BIP159) make it possible to run the full node in the pruned mode to save more disk space.
+
+Also, once the full node is synchronized to the network, you can downgrade the machine on Azure to take a less expensive one.
+
+Please check out [How to penny-pinch your Azure deployment](AzurePennyPinching.md).
+
+## What are the minimal requirements to run BTCPay?
+
+If you would like to run Bitcoin and Lightning Network nodes, the minimal requirements are :
+- 2GB Ram
+- 250 GB of storage (for the initial sync) can be pruned to 80GB.
+- Docker
+
+## How to install BTCPay Server?
+
+There are several videos and article online which you can check out:
+
+- [How to install BTCPay on Azure - guide](https://github.com/btcpayserver/btcpayserver-doc/blob/master/AzureDeployment.md)
+- [How to install BTCPay on Azure - video](https://www.youtube.com/watch?v=Bxs95BdEMHY)
+- [How to install BTCPay on LunaNode VPS - article](https://medium.com/@BtcpayServer/hosting-btcpayserver-on-lunanode-bf9ef5fff75b)
```

Comment:
You should add your channel and this one https://www.youtube.com/channel/UCpG9WL6TJuoNfFVkaDMp9ug